homeassistant/components/mqtt/models.py

```diff
@@ -187,6 +218,9 @@ class MqttData:
 
     client: MQTT | None = None
     config: ConfigType | None = None
+    debug_info: MqttDebugInfo = field(
+        default_factory=lambda: MqttDebugInfo(entities={}, triggers={})
+    )
```

Comment:
Would it make sense to split it?
```suggestion
    debug_entities_info: dict[str, EntityDebugInfo] = field(default_factory=dict)
    debug_trigger_info: dict[tuple[str, str], TriggerDebugInfo] = field(default_factory=dict)
```

Reply:
Yeah I realised that - but since you are already changing all the references to `hass.data[DATA_MQTT_DEBUG_INFO]["triggers"]` and `hass.data[DATA_MQTT_DEBUG_INFO]["entities"]` I think it still makes sense and it will make the final code easier to read.